scripts/SREP/example/script.sh

```diff
@@ -1,7 +1,24 @@
 #!/bin/bash
 
+# Source the logging system
+source ../LIBRARY/liblog.sh
+source ../LIBRARY/libshellcheck.sh
+
```

Comment:
Running this locally:
```
./script.sh     
./script.sh: line 4: ../LIBRARY/liblog.sh: No such file or directory
./script.sh: line 5: ../LIBRARY/libshellcheck.sh: No such file or directory
./script.sh: line 8: log_info: command not found
```
```suggestion
source ../../LIBRARY/liblog.sh
source ../../LIBRARY/libshellcheck.sh

```

Reply:
removed shell check will add in separate card, it works with liblog.sh 
```
./script.sh 
[2023-04-14 01:33:32 IST] [INFO] Getting pods in openshift-monitoring namespace 
NAME                                                     READY   STATUS      RESTARTS   AGE
alertmanager-main-0                                      6/6     Running     0          3d14h
alertmanager-main-1                                      6/6     Running     0          3d15h
cluster-monitoring-operator-64cb8c7fcf-g7zkv             2/2     Running     0          3d15h
configure-alertmanager-operator-fbcc556b5-rx7gx          1/1     Running     0          4h50m
configure-alertmanager-operator-registry-wqxq4           1/1     Running     0          4h51m
kube-state-metrics-7b4dc68565-f8ctw                      3/3     Running     0          3d14h
node-exporter-77wvr                                      2/2     Running     2          3d15h
node-exporter-9bpdj                                      2/2     Running     2          3d15h
node-exporter-d9ctx                                      2/2     Running     2          3d15h
node-exporter-d9xkl                                      2/2     Running     2          3d15h
node-exporter-dth6g                                      2/2     Running     2          3d15h
node-exporter-l7b4t                                      2/2     Running     2          3d15h
node-exporter-lqv2f                                      2/2     Running     2          3d15h
node-exporter-qk5pv                                      2/2     Running     2          3d15h
node-exporter-wxmsf                                      2/2     Running     2          3d15h
openshift-state-metrics-b559c99bf-kdfsh                  3/3     Running     0          3d14h
osd-cluster-ready-cvfln                                  0/1     Completed   7          3d15h
osd-rebalance-infra-nodes-28023570-vxqbl                 0/1     Completed   0          33m
osd-rebalance-infra-nodes-28023585-lk5ck                 0/1     Completed   0          18m
osd-rebalance-infra-nodes-28023600-4gl96                 0/1     Completed   0          3m34s
prometheus-adapter-7986c79c5d-k8r4k                      1/1     Running     0          2d15h
prometheus-adapter-7986c79c5d-z872j                      1/1     Running     0          2d15h
prometheus-k8s-0                                         6/6     Running     0          3d14h
prometheus-k8s-1                                         6/6     Running     0          3d15h
prometheus-operator-854b9d45d4-zsfgm                     2/2     Running     0          3d14h
prometheus-operator-admission-webhook-79b9999764-c5pgr   1/1     Running     0          3d15h
prometheus-operator-admission-webhook-79b9999764-vr5xh   1/1     Running     0          3d14h
sre-dns-latency-exporter-46mgs                           1/1     Running     1          3d15h
sre-dns-latency-exporter-5w4nb                           1/1     Running     1          3d15h
sre-dns-latency-exporter-669q6                           1/1     Running     1          3d15h
sre-dns-latency-exporter-c4p2j                           1/1     Running     1          3d15h
sre-dns-latency-exporter-dqpms                           1/1     Running     1          3d15h
sre-dns-latency-exporter-h7mds                           1/1     Running     1          3d15h
sre-dns-latency-exporter-jrmm7                           1/1     Running     1          3d15h
sre-dns-latency-exporter-jxt84                           1/1     Running     1          3d15h
sre-dns-latency-exporter-qhc48                           1/1     Running     1          3d15h
sre-ebs-iops-reporter-1-btqw7                            1/1     Running     0          3d14h
sre-stuck-ebs-vols-1-dgn92                               1/1     Running     0          3d14h
telemeter-client-7965b786c6-rcrns                        3/3     Running     0          3d14h
thanos-querier-cfcd6794d-mgm22                           6/6     Running     0          3d15h
thanos-querier-cfcd6794d-x6gcm                           6/6     Running     0          3d14h
token-refresher-6d8f85f497-9hm6c                         1/1     Running     0          3d14h
```
```
../../../../github/backplane-cli/ocm-backplane testjob get openshift-job-dev-7gbd9                       
INFO[0001] Using backplane Proxy URL: http://file.apac.redhat.com:3128 
TestId: openshift-job-dev-7gbd9, Status: Succeeded
```